tests/test_attributions_gemma.py

```diff
@@ -374,16 +376,19 @@ def verify_gemma_2_2b(s: str):
         verify_feature_edges(model, graph)
 
 
+@pytest.mark.skipif(not torch.cuda.is_available(), reason="CUDA not available")
```

Comment:
I think we should be able to run the small / large tests on the CPU as a part of CI. These are the main tests for circuit tracer so including them in the CI would be pretty valuable. We definitely need to skip the full gemma test though

Reply:
We shouldn't have to pull anything for these tests - the ones that uses the actual models should be skipped

Reply:
ah great, makes things easier